### PR TITLE
Enable SA1643: Destructor summary documentation

### PR DESCRIPTION
**Pull request by xtqqczze**
Enable SA1643: Destructor summary documentation should begin with standard text

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1643.md

### CODE REVIEW

File: src/System.Management.Automation/utils/CryptoUtils.cs [hunk 1]
```diff
@@ -628,7 +628,7 @@ protected void Dispose(bool disposing)
         }
 
         /// <summary>
-        /// Destructor.
+        /// Finalizes an instance of the <see cref="PSRSACryptoServiceProvider"/> class.
         /// </summary>
         ~PSRSACryptoServiceProvider()
         {
```
Comment (xtqqczze) [inline src/System.Management.Automation/utils/CryptoUtils.cs:634]: I wonder why we call `Dispose(true)` not `Dispose(false)` in the finalizer.

**Comment by xtqqczze** [comment]
@iSazonov Maybe we can remove the finalizer on classes without unmanaged resources?

File: src/System.Management.Automation/utils/CryptoUtils.cs (hunk 1 above)
Comment (iSazonov) [reply src/System.Management.Automation/utils/CryptoUtils.cs:634]: Please open new PR for discussion.

**Review by iSazonov — APPROVED** [review]

File: src/System.Management.Automation/utils/CryptoUtils.cs (hunk 1 above)
Comment (xtqqczze) [reply src/System.Management.Automation/utils/CryptoUtils.cs:634]: Submitted #14246.

**Comment by iSazonov** [comment]
@xtqqczze Please resolve merge conflicts.

**Comment by xtqqczze** [comment]
Depends on #14248.

**Pull request merged**